packages/venia-ui/lib/components/CartPage/ProductListing/product.js

```diff
@@ -95,7 +92,9 @@ export const REMOVE_ITEM_MUTATION = gql`
         removeItemFromCart(input: { cart_id: $cartId, cart_item_id: $itemId }) {
             cart {
                 id
+                ...CartPageFragment
```

Comment:
When performing a mutation within the `CartPage` we can just refetch the entire composed fragment rather than requiring each component to fetch specifics. This may result in "overfetch" such as in the case of adding a coupon where you only really need to update the total and discounts.

Reply:
I'm fine with this, under the assumption that over fetching doesn't causes unnecessary re-renders if that data didn't change.

Reply:
I'll have to look -- I'm not sure, it seems super fast.

Reply:
For now I left the fetch policy so this doesn't matter.